src/main/windows/help.ts

```diff
@@ -0,0 +1,94 @@
+import { app as rawApp, BrowserWindow, Menu, shell } from 'electron'
+import { appIcon } from '../application-constants'
+import { getLogger } from '../../shared/logger'
+import { ExtendedAppMainProcess } from '../types'
+import { join } from 'path'
+import { pathExists } from 'fs-extra'
+import { appWindowTitle } from '../../shared/constants'
+
+const log = getLogger('main/help')
+const app = rawApp as ExtendedAppMainProcess
+
+async function getHelpFileForLang(locale: string) {
+  const appPath = app.getAppPath()
+
+  const contentFilePath = join(appPath, `/html-dist/help/${locale}/help.html`)
+  if (await pathExists(contentFilePath)) {
+    return contentFilePath
+  } else {
+    log.warn(
+      `Did not found help file for language ${locale}, falling back to english`
+    )
+    return join(appPath, `/html-dist/help/en/help.html`)
+  }
+}
+
+let win: BrowserWindow | null = null
+
+export async function openHelpWindow(locale: string) {
+  if (win) {
+    win.focus()
+    return
+  }
+
+  log.debug('open help', locale)
+  const defaults = {
+    bounds: {
+      width: 500,
+      height: 638,
+    },
+    headerHeight: 36,
+    minWidth: 450,
+    minHeight: 450,
+  }
+  win = new BrowserWindow({
+    backgroundColor: '#282828',
+    darkTheme: true, // Forces dark theme (GTK+3)
+
+    icon: appIcon(),
+    minHeight: defaults.minHeight,
+    minWidth: defaults.minWidth,
+    show: false,
+    title: appWindowTitle + '-' + app.translate('menu_help'),
```

Comment:
i think, this is mean as sort of hierarchy, as in the main window. in this case, we should use spaces around the minus character, so `DeltaChat - Help` (as in the main window that reads `DeltaChat - me@example.org`)